frontend/src/components/common/button/button.tsx

```diff
@@ -0,0 +1,42 @@
+import clsx from 'clsx';
+import { ButtonNames } from 'common/enums';
+import * as React from 'react';
+import styles from './styles.module.scss';
+
+interface ButtonTypes {
+  [ButtonNames.SIGN_IN]: string;
+  [ButtonNames.SIGN_IN_GOOGLE]: string;
+  [ButtonNames.APPOINTMENT]: string;
+  [ButtonNames.COVID]: string;
+  [ButtonNames.READ_ALL]: string;
+}
+
+const BUTTON_TEXT: ButtonTypes = {
+  [ButtonNames.SIGN_IN]: 'Sign in',
+  [ButtonNames.SIGN_IN_GOOGLE]: 'Sign in with Google',
+  [ButtonNames.APPOINTMENT]: 'Make an appointment',
+  [ButtonNames.COVID]: 'COVID 19 Help',
+  [ButtonNames.READ_ALL]: 'Read all reviews',
+};
+
+interface ButtonType {
+  btnType: keyof ButtonTypes;
+  fill: boolean;
+  border: boolean;
+  onClick(): void;
+}
```

Comment:
```suggestion
interface Props {
  type: ButtonType;
  styleType: ButtonStyleType;
  color: ButtonColor;
  label: string;
  onClick(): ButtonClickEvent;
}
```
We need to add 2 enums `ButtonStyleType` and `ButtonColor`. And base on this props - we should to set properly classes
ButtonType is the 'enum' with button types(`button`, `submit` and etc)

Reply:
As I understood, I don't need to add `ButtonType` enum as I should do with enums `ButtonStyleType` and `ButtonColor`. Is it some build-in enum? I thought so, but then I got error "Cannot find name 'ButtonType'."
And the same error for `ButtonClickEvent`.